### PR TITLE
arduino json 6 async web server migration

### PR DESCRIPTION
**Pull request by baggior**
fix for ArduinoJson v. 6

### CODE REVIEW

**Comment by baggior** [comment]
#487

**Comment by baggior** [comment]
please 
can somebody fix the travis errors?
They aren't realted to the code I've changed but I suppose is a matter of travis configuration itself...

**Comment by baggior** [comment]
I've also fixed the travis integration problem

@me-no-dev please MERGE .. thankyou

**Comment by baggior** [comment]
@me-no-dev could you merge please?

File: src/AsyncJson.h [hunk 1]
```diff
@@ -35,8 +35,15 @@
 #ifndef ASYNC_JSON_H_
 #define ASYNC_JSON_H_
 #include <ArduinoJson.h>
+#include <Print.h>
 
-constexpr char* JSON_MIMETYPE = "application/json";
+#if ARDUINOJSON_VERSION_MAJOR == 5
+  #define ARDUINOJSON_5_COMPATIBILITY
+#else
+  #define DYNAMYC_JSON_DOCUMENT_SIZE  1024
```
Comment (atanisoft) [inline src/AsyncJson.h]: perhaps fix the typo to : DYNAMIC_JSON_DOCUMENT_SIZE

Comment (baggior) [reply src/AsyncJson.h]: I agree

**Comment by baggior** [comment]
@me-no-dev could you merge please?

**Comment by me-no-dev** [comment]
alright :) merged :) sorry guys... really have not had time in a while to look at my private projects

**Comment by proddy** [comment]
Could we also update some of the examples in the README too, for example the web socket [example](https://github.com/me-no-dev/ESPAsyncWebServer#direct-access-to-web-socket-message-buffer) will not work with ArduinoJson6.

**Pull request merged**